Code/Makefile

```diff
@@ -10,6 +10,9 @@ debug: elf_debug
 release: elf
 release_oe: BUILD_OE = 1
 release_oe: prx
+debug_oe: BUILD_OE = 1
+debug_oe: CONFIG = DEBUG
+debug_oe: prx_debug
```

Comment:
While I sympathize with the idea of adding a debug_oe, there are some other changes that need to happen to truly support this in the Makefiles.
I added the BUILD_OE as a very quick hack to get the slim version out; I can go back and rework it tho, if this is of interest..